models/en-US.json

```diff
@@ -4,21 +4,34 @@
             "invocationName": "hello buttons",
             "intents": [
                 {
-                    "name": "AMAZON.CancelIntent",
+                    "name": "HelloIntent",
+                    "slots": [],
                     "samples": [
-                        "cancel"
+                        "hello"
                     ]
                 },
+                {
+                    "name": "AMAZON.FallbackIntent",
```

Comment:
doesn't this require some code changes, to support the FallbackIntent in the code?